src/services/fetchers/CoingeckoPriceMultiFetcher.ts

```diff
@@ -89,20 +84,16 @@ export default class CoingeckoPriceMultiFetcher {
       throw new Error(response.data.Message);
     }
 
-    const outputs: OutputValues[] = [];
+    const outputs: NumberOrUndefined[] = [];
 
     inputs.forEach((input) => {
       const {id, currency} = input;
-      const value: number | undefined = (response.data[id] || {})[currency.toLowerCase()];
+      const value: NumberOrUndefined = (response.data[id] || {})[currency.toLowerCase()];
 
       if (value) {
         this.logger.debug(`[CoingeckoPriceMultiFetcher] resolved price: ${id}-${currency}: ${value}`);
 
-        outputs.push({
-          id,
-          currency,
-          value: value,
-        });
+        outputs.push(value);
```

Comment:
this is a bug. this fetcher returns any combination of `id` and `currency`.

Reply:
Returning `id` and `currency` was used to sort the outputs:

https://github.com/umbrella-network/pegasus/blob/e71fdb0636c30df865e9ce07baeb989e956d8ae3/src/services/feedProcessors/CoingeckoMultiProcessor.ts#L78-L84

Now we don't need to do this at the fetcher level, but at the multi processor level.

Reply:
when you call for `http://...id=BTC...currency=USDT...` API can return eg `BTC-USDC`, `BTC-USDT`, `ETH-USDT`... so I want you save them all to DB under `id-currency` for now.